### DISCUSSION TITLE
Is there a way to disable AutoComplete in ArgoCD  fields?

### DISCUSSION THREAD

**Discussion by vkumar85** [argoproj/argo-cd#9766] [Q&A]
During our Vulnerability Scan, we are flagged an issue with ArgoCD that AutoComplete not Disabled for Sensitive Data Fields.

AutoComplete field should be disabled for any field that contains or could contain sensitive data, such as passwords, SIN, credit card numbers, etc. If the user is accessing the website on a share device or web browser then it is possible that another user could navigate to the website and auto populate the field with the previous user’s password.

Do we have configuration or start-up switch to disable AutoComplete?

**Comment by crenshaw-dev** [comment]
@vkumar85 that configuration doesn't currently exist, but I'd welcome a PR to add it! If you don't have time for a PR, then an Issue would still be appreciated.

**Comment by jannfis** [comment]
Do you have a list of fields where this would apply?

**Comment by vkumar85** [comment]
@jannfis Sorry for the late response. I currently don't have the list of fields but I am working to get the list of fields , which the pen test organization found to be vulnerable, and I will post it here as soon as I get it.
